x/group/server/msg_server.go

```diff
@@ -457,6 +461,33 @@ func (s serverImpl) CreateProposal(ctx types.Context, req *group.MsgCreatePropos
 		return nil, err
 	}
 
+	// Try to execute proposal immediately
+	if req.Exec == group.Exec_EXEC_TRY {
+		// Consider proposers as Yes votes
+		for i := range proposers {
+			ctx.GasMeter().ConsumeGas(gasCostPerIteration, "vote on proposal")
+			_, err = s.Vote(ctx, &group.MsgVoteRequest{
+				ProposalId: id,
+				Voter:      proposers[i],
+				Choice:     group.Choice_CHOICE_YES,
+			})
+			// Should we return MsgCreateProposalResponse with proposal id?
```

Comment:
I am also in same dilemma, may be @amaurym or @robert-zaremba can suggest here.

Reply:
IMO we should - the proposal has been created so it would be good to return it. Maybe we should also consider wrapping the error with some more description - e.g. `The proposal was created but failed on [vote/exec error]. `